lms/djangoapps/grades/event_utils.py

```diff
@@ -0,0 +1,49 @@
+from openedx_events.learning.data import (
```

Comment:
@wowkalucky @andrii-hantkovskyi This can be placed inside existing `grades/events.py` module. No need creating utils specific one